assets/js/main.js

```diff
@@ -49,60 +42,43 @@ var getClickEvent = function () {
   return 'ontouchstart' in document.documentElement ? 'touchend' : 'click';
 };
 
-/**
- * Opens and closes the mobile menu
- */
-var toggleMenu = function () {
-  $menuToggle.toggleClass('open');
-  $navigation.slideToggle(250);
-};
-
-$menuToggle.on(getClickEvent(), toggleMenu);
-
-// When a mobile nav link is clicked we should close the menu
-$('.header-nav-list-item').on(getClickEvent(), function () {
-  // We should be checking some state here instead of checking the width
-  if ($window.width() <= 770) {
-    toggleMenu();
-  }
-});
 
 $window.resize(function () {
-  // quick copy/paste, can be revisited for betterness
-
-  // brings back the nav if someone resizes the page
-  // needs work, still breaks with several click/resize combos
-  // probably unlikely that users will break it but should be fixed.
-  if ($(this).width() > 770 && $navigation.is(':hidden')) {
-    $navigation.show();
-  }
-
   // refetch transition heights on resize as well
   getTransitionHeights();
 }).scroll(function () {
   var s = window.scrollY;
 
-  if (s >= splashTransitionHeight - buffer && s < secondTransitionHeight) {
+  if (s >= splashTransitionHeight && s < secondTransitionHeight) {
```

Comment:
I don't think these first 2 if blocks are needed. They are all 3 adding the same class to the same element.

I think `s >= thirdTransitionHeight` would cover the first 2 cases.

```
.scroll(function () {
  if (window.scrollY >= thirdTransitionHeight) {
    $colorChangeElements.addClass('after-fold');
  }
  else {
    $colorChangeElements.removeClass('after-fold');
  }
}
```

which also means you wouldn't need the first two heights stashed. Those 3 could just become `transitionHeight` and then your `getTransitionHeights` could be something like:

```
var setTransitionHeight = function () {
  transitionHeight = $('.big-splash').height() + $('#who-we-are').height() + $('#what-we-do').height();
};
```